api-specificatie/Bevraging-Historie/openapi.yaml

```diff
@@ -347,29 +334,36 @@ paths:
             type: string
             maxLength: 9
             minLength: 9
+        - in: header
+          name: api-version
+          description: "Bevat de versie van de aan te roepen API conform Landelijke API-strategie."
+          required: false
+          schema:
+            type: string
+            example: 1.0.1
         - in: query
           name: fields
-          description: "Geeft de mogelijkheid de inhoud van de body van het antwoord naar behoefte aan te passen. Bevat een door komma's gescheiden lijst van veldennamen. Als niet-bestaande veldnamen worden meegegeven wordt een 400 Bad Request teruggegeven. Wanneer de parameter fields niet is opgenomen, worden alle gedefinieerde velden die een waarde hebben teruggegeven. Zie [functionele specificaties](https://github.com/VNG-Realisatie/Bevragingen-ingeschreven-personen/blob/master/features/fields.feature)"
+          description: "Geeft de mogelijkheid de inhoud van de body van het antwoord naar behoefte aan te passen. Bevat een door komma's gescheiden lijst van veldennamen. Als niet-bestaande veldnamen worden meegegeven wordt een 400 Bad Request teruggegeven. Wanneer de parameter fields niet is opgenomen, worden alle gedefinieerde velden die een waarde hebben teruggegeven. Zie functionele specificaties)"
           required: false
           schema:
             type: string
         - in: query
           name: geldigop
-          description: "De peildatum van de periode waarover de verblijfstitelhistorie wordt opgevraagd. Het resultaat bevat de verblijfstitelhistorie waarvan de peildatum op of na de ingangsdatumVerblijfstitel ligt en op of voor de datumEindeVerblijfstitel ligt."
+          description: "De peildatum van de periode waarover de verblijfstitelhistorie wordt opgevraagd. Het resultaat bevat de verblijfstitel van de persoon op het moment van de peildatum."
           required: false
           schema:
             type: string
             format: date
         - in: query
-          name: geldigtotenmet
-          description: "De datum waarop de verblijfstitel zijn geldigheid verliest De einddatum van de periode waarover de verblijfstitelhistorie wordt opgevraagd. Het resultaat bevat de verblijfstitelhistorie waarvan de datumEindeVerblijfstitel op of voor datumTotEnMet ligt. Dit attribuut wordt uitsluitend opgenomen indien er sprake is van een verblijfstitel voor bepaalde tijd. Bij een verblijfstitel voor het leven komt het niet voor."
+          name: geldigtot
+          description: "De datum waarop de verblijfstitel zijn geldigheid verliest De einddatum van de periode waarover de verblijfstitelhistorie wordt opgevraagd. Het resultaat bevat de verblijfstitels waarvan de datumEindeVerblijfstitel op of voor geldigTot ligt. Dit attribuut wordt uitsluitend opgenomen indien er sprake is van een verblijfstitel voor bepaalde tijd. Bij een verblijfstitel voor het leven komt het niet voor."
```

Comment:
> De datum waarop de verblijfstitel zijn geldigheid verliest

Dit is denk ik geen juiste omschrijving. Het gaat om de einddatum van de periode waarover men de verblijfstitels wil raadplegen

Reply:
Correcte omschrijving ingevoegd